screens/landing-page/landing-page.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home Page</title>
+    <link
+      rel="stylesheet"
+      href="https://loremui-library.netlify.app/css/components.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/1d3a2968ad.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="landing-page.css" />
+    <link rel="stylesheet" href="/css/color.css" />
+    <link rel="stylesheet" href="/css/navbar.css" />
+    <link rel="stylesheet" href="../product/product.css" />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16X16"
+      href="/assets/cricket.png"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-heading p-1">
+        <h1 class="home-heading">Crickart</h1>
+        <div class="search-bar">
+          <input type="text" class="input-basic px-2" placeholder="Search" />
+          <div class="search-button">
+            <a href="#"><i class="fa fa-search search-icon"></i></a>
+          </div>
+        </div>
+        <div class="nav-items">
+          <a class="px-05 fs-2" href="../login/login.html">
+            <p>Login</p>
+          </a>
+          <div class="badge-icon">
+            <a class="px-05" href="wishlist.html">
+              <i class="fa fa-heart fa-2x"></i>
+              <span class="label label-cart fs-1">3</span>
+            </a>
+          </div>
+          <div class="badge-icon">
+            <a class="px-05" href="/screens/cart/cart.html">
+              <img src="/assets/shopping-cart.png" alt="shopping-cart" />
+              <span class="label label-cart fs-1">6</span>
+            </a>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <div class="brand-banner">
+        <img src="/assets/Landing-page.png" alt="" class="banner" />
+        <div class="search-wrapper">
+          <p class="banner-heading">
+            All your favorite cricket equipment at affordable rates
+          </p>
+          <a href="/screens/product/product.html" class="shop-link">Buy Now</a>
+        </div>
+      </div>
+      <p class="category-heading">Category of Equipment</p>
+      <div class="category-wrapper">
+        <div class="category-bats">
+          <img src="/assets/GN-bat(1).jpg" alt="" class="equipment-category" />
+          <p class="category-text">Bats</p>
+        </div>
+        <div class="category-balls">
+          <img
+            src="/assets/White-ball.jpeg"
+            alt=""
```

Comment:
You could give value to the `alt` attribute here as well as at every other `<img />` as that helps for accessibility.

Reply:
Thanks for pointing out and giving time for  last minute reviews. Thank you @VishalPatil18!!